examples/jenkins/Linux.Jenkinsfile

```diff
@@ -11,8 +11,8 @@ pipeline {
     // Artifacts Folder
     ArtifactsFolder = "Artifacts"
     // LifeTime Specific Variables
-    LifeTimeHostname = 'lifetime.acmecorp.com'
     LifeTimeAPIVersion = 2
+    LifeTimeEnvironmentURL = 'lifetime.acmecorp.com'
```

Comment:
Rename variable to "LifeTimeHostname" to keep consistency with Windows version.

Reply:
Variable renamed